sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.2.5 (2021-03-31)
```

Comment:
@sadasant - this date was probably too ambitious on my part. Perhaps tomorrow instead?

Reply:
Tomorrow makes more sense to me.

Reply:
Fixed.